packages/integrations/scripts/buildBundles.ts

```diff
@@ -0,0 +1,22 @@
+import { spawnSync } from 'child_process';
+import { readdirSync } from 'fs';
+import { join } from 'path';
+
+function getIntegrations(): string[] {
+  // The index file is only there for the purposes of npm builds (for the CDN we create a separate bundle for each
+  // integration) and the flags file is just a helper for including or not including debug logging, whose contents gets
+  // incorporated into each of the individual integration bundles, so we can skip them both here.
+  return readdirSync(join(__dirname, '..', 'src')).filter(file => !file.endsWith('index.ts'));
+}
+
+for (const integration of getIntegrations()) {
+  for (const jsVersion of ['ES5', 'ES6']) {
```

Comment:
I wonder if we can run this async on both es5 and es6 at the same time.

Reply:
We probably also need to check for non-zero exit code too...

Reply:
yeah - thats probably the tricky part. Would be interesting to try out here!

Reply:
Oh, its about 5x quicker!

Reply:
alright I'm in favour of shipping that - but we'll need to make sure to have some kind of flag that can bring back the sync behavior if we need to debug.